src/sentry/services/hybrid_cloud/project/impl.py

```diff
@@ -72,3 +76,32 @@ def serialize_many(
             user=as_user,
             serializer=ProjectSerializer(),
         )
+
+    def create_project_for_organization(
+        self,
+        *,
+        organization_id: int,
+        project_name: str,
+        platform: str,
+        user_id: int,
+        add_org_default_team: Optional[bool] = False,
+    ) -> RpcProject:
+        with transaction.atomic(router.db_for_write(Project)):
+            project = Project.objects.create(
+                name=project_name,
+                organization_id=organization_id,
+                platform=platform,
+            )
+
+            if add_org_default_team is not None:
```

Comment:
```suggestion
            if add_org_default_team:
```
We want to skip this if it's `False` too, right?